hw3/01-characters.js

```diff
@@ -1,4 +1,51 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+let app = document.querySelector("#results");
+let row = document.querySelector(".row");
+
+const charactersToDOM = (chars) => {
+	//character card
+	let card = document.createElement("div");
+	card.classList.add("figure", "col-md-3", "col-6");
+
+	//character image
+	let imageContainer = document.createElement("div");
+	imageContainer.classList.add(
+		"d-flex",
+		"justify-content-center",
+		"align-items-center"
+	);
+	let image = document.createElement("img");
+	image.src = chars.imageUrl;
+	image.alt = `picture of ${chars.image}`;
```

Comment:
In the alt text it is not necessary to add phrases such as "picture of", “image of” or “graphic of”.